README.md

```diff
@@ -91,6 +91,14 @@ To enable message feedback, you will need to set up CosmosDB resources. Then spe
 /.env
 - `AZURE_COSMOSDB_ENABLE_FEEDBACK=True`
 
+#### Local Setup: Enable SQL Server
+To enable SQL Server, you will need to set up SQL Server resources. Then specify these additional environment variables:
+- `DATASOURCE_TYPE`
```

Comment:
Can we add a note here that the value of `DATASOURCE_TYPE` needs to be equal to `AzureSqlServer`?